content/blog/prioritizing-the-grind/index.md

```diff
@@ -0,0 +1,111 @@
+
+Oftentimes a task will come across a team that is obviously suited for an individual with a skillset that matches up perfectly with it. Potentially the most efficient decision would be to assign it to that person and let it get done quickly. And the next time, and the time after that. This works until that person is out sick, or has left the company. Perhaps this is a skill some or all of the others on the team want to learn, whether it's for personal / career development, or just for fun. Sometimes it's the opposite, no one wants to do it because of the drudgery or complexity. Having more people level up on work that interest them, or share the burden of something no one wants to do always yields vastly improved results.
+
+In a teacher/student environment we have a win-win situation. Teaching someone is not only rewarding on its own, but the thing you're teaching is further solidified with a deeper understanding while you're teaching it. Someone learning something new and fulfilling is a powerful way to improve their morale while increasing the number of people that have that skill on your team.
+
+If before you had an individual quietly fixing something that constantly breaks, or needs manual intervention regularly, you can break this cycle by adding more people to the process. The person that held it all together gets help, sometimes this can prevent burnout or employee retention in a serious way. Bringing more people into a situation like this can also give you unexpected results: a process that wasn't automated gets automated away because fresh eyes see a similar problem from the past.
+
+Fostering this environment will empower your team to continue this collaboration without being told to do so. People will automatically jump in to help those that need it, or offer their support to teach those that want to learn. This starts to become a fun place to go to work every day. It creates a cycle of more capable and happy individuals which leads to the feeling of rampant creation anyone that's been on one of these teams will recognize immediately.
+
+There is an upper limit to how expensive collaboration is on a given team. Communication scales exponentially for every new person on a team which can cause everything to grind to a halt. There are also best judgement calls you will make regularly about critical fires that come up that take precedence, as long as this is not the norm this is totally normal. Efficiency is important, but not if it kneecaps your team every day away from what will help them grow and work on what they feel is important. 
```

Comment:
Learning how to find the upper limit, or how to notice when you're approaching it is a tough skill to learn. Do you have "flags" that you watch for? Could those be made into a pseudo code?

```
while(
engagement && perceivedProductivity &&
learning &&
deliveringValue 
) 
```

Or 🤷‍♂️

Reply:
Good question, I'll think of ways to talk more practically about it.